FRUI/FRUI/ViewControllers/AuthStepViewController.swift

```diff
@@ -126,7 +128,36 @@ class AuthStepViewController: UIViewController {
                 //  Set auth callbacks in list for rendering
                 self.currentNode = node
                 self.authCallbacks = node.callbacks
-                self.renderAuthStep()
+                
+                //  If DeviceProfileCallback is the only Callback, then immediately collect, and submit
+                if self.authCallbacks.count == 1, let deviceProfileCallback = self.authCallbacks.first as? DeviceProfileCallback {
+                    self.loadingView.loadingText = deviceProfileCallback.message
```

Comment:
I was never able to see the configured message in AM showing up with the spinner, although I see that the loadingText is set with correct value, etc... What I see is the spinner with with message "Loading..."

Reply:
@spetrov I've updated the loading indicator to better visualize the situation. Please pull the latest code, and let me know if that works better for you :)